pending/exports-and-boilerplate.pod

```diff
@@ -0,0 +1,116 @@
+=encoding utf8
+
+=head1 Reducing boilerplate and managing exports in Dancer2
+
+Dancer makes it easy to start small and grow fast. Today we will be looking at a techique to help manage that growth.
+
+A typical mature Dancer2 app will span multiple packages and each package will make use of Dancer's plugins and other modules available on CPAN.
+
+  use Scalar::Util qw< blessed >;
+  use List::Util qw< first all none sum >;
+  use Dancer2 appname => MyApp;
+  use Dancer2::Plugin::DBIC qw< schema resultset >;
+  use Dancer2::Plugin::Redis;
+
+After the first couple of files, the copy-pasting becomes tedious and if you later change your mind about what to include, you leave yourself open to runtime errors when you try to call a function you never imported into the package namespace.
```

Comment:
I would end the sentence at "tedious". Then the rest is another sentence. I read this paragraph three times until I realized it was one long sentence. I kept getting confused. :)